README.md

```diff
@@ -197,6 +197,63 @@ class Serializer(serializers.Serializer):
 The implementation of `inline_serializer` can be found in `utils.py` in this repo.
 
 
+## Exception Handling
+
+### Raising Exceptions in Services
+Now we have a great separation between business logic and HTTP logic. The business logic lives in the services and the HTTP logic lives in the APIs.
+
+In order to keep this separation of concerns our services must not use the `rest_framework.exception` classes because they are bounded with HTTP status codes. 
```

Comment:
Not really sure why are we excluding `rest_framework.exception`, but including `django.core.exceptions` :thinking: 

Should be either - you can use both or use None :thinking: 

Sometimes it's handy to raise `ValidationError` or use the `get_object_or_404` util :thinking: 

Lets discuss

Reply:
Up for discussion on Monday.